docs/content/en/docs/troubleshooting/troubleshooting.md

```diff
@@ -415,6 +517,12 @@ govc vm.power -off -force $VM_NAME
 govc object.destroy $VM_NAME
 ```
 
+### Restore VM for machine in vSphere
+
+When a CAPI machine is in not ready state, it is possible that the machine object does not match the underlying VM in vSphere. Instead, the machine is pointing to a VM that does not exist anymore.
```

Comment:
Does "no ready" appear literally? If so, perhaps mark it `not ready` or `Not Ready`, depending on how it appears. If you just mean that as a general state of things, then the text is fine.

Reply:
yes i mean more in general